module/submissions/helper.js

```diff
@@ -148,6 +148,11 @@ module.exports = class SubmissionsHelper {
 
                     // Push new submission to kafka for reporting/tracking.
                     this.pushInCompleteSubmissionForReporting(submissionDocument._id);
+
+                    if( submissionDocument.projectId && submissionDocument.taskId ) {
```

Comment:
@aman-tunerlabs - Instead of using these 2 keys, use one key - "referenceFrom" which value is project can be pushed ti kafka